src/view/components/extensionView.jsx

```diff
@@ -74,7 +74,11 @@ const ExtensionView = ({
     // (which were set during submitForm()) to see if the form is valid.
     // https://github.com/jaredpalmer/formik/issues/1580
     formikPropsRef.current.setSubmitting(false);
-    return Object.keys(formikPropsRef.current.errors).length === 0;
+
+    // Since React18, errors are not up to date on the formikPropsRef.current object.
+    // We collect the errors by running imperative validateForm() method.
+    const errors = await formikPropsRef.current.validateForm();
```

Comment:
Since React18, it seems the errors on the formikRef are no longer up to date in the same render cycle. So I was forced to call the validateForm imperatively in order to get the form errors in the same render cycle.